modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPhpCodegen.java

```diff
@@ -441,6 +453,13 @@ public String toVarName(String name) {
         // sanitize name
         name = sanitizeName(name); // FIXME: a parameter should not be assigned. Also declare the methods parameters as 'final'.
 
+        return name;
+    }
+
+    @Override
+    public String toVarName(String name) {
+        name = toNonConventionVarName(name);
```

Comment:
looks like this will break the nameMapping feature as the mapped name won't be used directly anymore.

Reply:
Yes you are right. There should nothing happen if the name is mapped. I will fix that.

Reply:
I have no better idea to combine the logic better instead of returning the nameMapping in the first place.
The main problem was that in the past the getterAndSetterCapitalize was calling toVarName and there was already some formatting happening so i have to keep this logic to not break older stuff.
Suggestions are welcome. ;-) Else it should not breaking nameMapping anymore. :-)